tools/ci/src/ci.rs

```diff
@@ -59,28 +82,23 @@ impl CI {
     }
 
     fn prepare<'a>(&self, sh: &'a xshell::Shell) -> Vec<PreparedCommand<'a>> {
-        let mut flags = Flag::empty();
-
-        if self.keep_going {
-            flags |= Flag::KEEP_GOING;
-        }
-
+        let args = self.into();
```

Comment:
(Nit): A type annotation here would be very useful when reading this code, especially since `into()` is quite general.

```rust
let args: Args = self.into();
```

While not necessary to compile the code (or if you have Rust Analyzer), it's nice when reviewing the code in Github.